src/cr/cube/stripe/insertion.py

```diff
@@ -61,6 +61,15 @@ class SumSubtotals(_BaseSubtotals):
     bases are additive even across subtrahends of subtotals.
```

Comment:
Can you please also document in this docstring what the `diffs_nan` is used for? I can't figure it out by inspecting the changes. I can guess that its the case where you "don't want to add the specific measure values for the differences", but even so, I'd like to know when that applies and why.